README.md

```diff
@@ -13,6 +13,16 @@ npm install --save redux-actions
 import { createAction, handleAction, handleActions } from 'redux-actions';
 ```
 
+## Note
+
+We are also supporting a UMD build:
```

Comment:
that looks really nice, another option would be [cndjs][https://cdnjs.com/]... I would go for both 😅
I will check that 👍